README.md

```diff
@@ -1,93 +1,110 @@
 # Prototype repo for new qt toolkit
 
 ## Introduction
-This toolkit is an open source resource inetnded for use with the ArcGIS Runtime
+
+This toolkit is an open source resource intended for use with the ArcGIS Runtime
 SDK for Qt. It provides ready made UI components to complement mapping and GIS
 applications. Because it is open source you are encouraged to modify these tools
 and add your own.
 
 ## Structure
+
 This repo contains 3 toolkit projects, each intended for use with a different
 flavor of the ArcGIS Runtime SDK for Qt:
 
 - toolkitqml (for use with QML only apps)
-- toolkitquick (for use with C++ apps which use a QtQuick UI)
+- toolkitcpp (for use with C++ apps which use a QtQuick UI)
 - toolkitwidgets (for use with C++ apps which use a QWidgets UI)
 
 You need to clone this repository locally to incorporate it into your own
 project.
 
+### toolkitqml.pri
 
-### toolkitqml
+#### importing the toolkit (toolkitqml.pri)
 
-#### importing the toolkit
 A good way to start using the toolkit is to use one of the template apps which
 get added to QtCreator when you install the ArcGIS Runime SDK for Qt.
 
-- In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Runtime 100.6
+- In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Runtime 100.X
   Quick QML App
 - choose settings to match the platform etc. you are building for
 - in the .pro` file of your new app, add a line to add the toolkit for your
   QML application. For example:
 
+```pro
+include(path/to/toolkitqml.pri)
 ```
-include(path/to/toolkit/toolkitqml.pri)
-```
 
-#### using a tool from the toolkit
+#### using a tool from the toolkit (toolkitqml.pri)
+
 Once you have successfully imported the toolkit, you can access individual tools
 in your own QML files.
 
 - add an import statement for the toolkit:
-  `import esri.arcgisruntime.toolkit 1.0`
+  `import "qrc:///Esri/ArcGISRuntime/Toolkit" as Toolkit`
```

Comment:
I wonder if it might be better to promote a name which includes Esri (and to avoid clashes with anything else called toolkit)

 `import "qrc:///Esri/ArcGISRuntime/Toolkit" as ArcGISRuntimeToolkit`